src/firefly/js/templates/lightcurve/LcConverterFactory.js

```diff
@@ -60,15 +104,19 @@ export function getAllConverterIds() {
     return Object.keys(converters);
 }
 
-export function getConverter(datasetConverterId) {
-    const converter = datasetConverterId && converters[datasetConverterId];
+export function getConverter(converterId) {
+    const converter = converterId && converters[converterId];
     if (!converter) {
-        logError(`Unable to find dataset converter ${datasetConverterId}`);
+        logError(`Unable to find dataset converter ${converterId}`);
         return;
     }
     return converter;
 }
 
+export function getMissionName(converterId) {
+    const converterData = converterId && converters[converterId];
+    return get(converterData, 'missionName', converterId);
```

Comment:
should convertId to be capitalized? like get(converterData, 'missionName', convertId.toUpperCase())?